TOC-tidb-cloud.md

```diff
@@ -294,6 +294,7 @@
 - Reference
   - TiDB Cluster Architecture
     - [Overview](/tidb-architecture.md)
+    - [TSO](/tso.md)
```

Comment:
Should it really be an own topic in the TiDB Cluster Architecture? I think it may be better placed in either the `Overview`/`tidb-architechture.md` or in Reference/SQL/Transactions.

Reply:
I think it is better to be separate.

Reply:
Can we place this document after [Scheduling](/tidb-scheduling.md)? For TiDB users, they might want to learn about the storage, computing, and scheduling architecture info of TiDB first?

Reply:
@qiancai Yes that's fine with me.